test/loadTest/README.md

```diff
@@ -53,11 +53,11 @@ Properties:
 		* THREADS        - number of concurrent threads (default: 1)
 		* RAMP_UP        - the amount of time for creating the total number of threads (default: 1)
 	    * TEST_TIME      - test duration time in seconds (default:30)
-		* SERVICE        - service header (default: update_ngsiv2)
+		* SERVICE        - service header (default: )
```

Comment:
I'd suggest: "(default: no service is used, i.e. Fiware-Service header is omitted)". I think it would be clearer.